packages/cdktf-cli/templates/csharp/.hooks.sscaff.js

```diff
@@ -0,0 +1,64 @@
+const { execSync } = require('child_process');
+const { readFileSync, writeFileSync } = require('fs');
+const os = require('os');
+
+const cli = require.resolve('../../bin/cdktf');
+
+exports.pre = (variables) => {
+  try {
+    if (os.platform() === 'win32') {
+      execSync('where dotnet')
+    }
+    else {
+      execSync('which dotnet')
+    }
+  } catch {
+    console.error(`Unable to find "dotnet" installation. Install from https://dotnet.microsoft.com/download`);
+    process.exit(1);
+  }
+};
+
+exports.post = options => {
+  const { nuget_cdktf, cdktf_version } = options;
+  if (!nuget_cdktf) {
+    throw new Error(`missing context "nuget_cdktf"`);
+  }
+
+  // Terraform Cloud configuration settings if the organization name and workspace is set.
+  if (options.OrganizationName != '') {
+    console.log(`\nGenerating Terraform Cloud configuration for '${options.OrganizationName}' organization and '${options.WorkspaceName}' workspace.....`)
+    terraformCloudConfig(options.$base, options.OrganizationName, options.WorkspaceName)
+  }
+
+  pkgFileName = nuget_cdktf.substring(nuget_cdktf.lastIndexOf('/') + 1);
+  pkgName = pkgFileName.substring(0, pkgFileName.lastIndexOf(".".concat(cdktf_version)));
+  srcFolder = nuget_cdktf.substring(0, nuget_cdktf.lastIndexOf('/'));
```

Comment:
Might be better to use `path` to split rather than checking for `/` which is likely not cross-platform safe. There's also the option of explicitly passing other variables.

Reply:
I'm also pretty sure these will fail when a version number is passed for `nuget_cdktf`.